### DISCUSSION TITLE
Copilot cannot be authorized.

### DISCUSSION THREAD

**Discussion by ggggarlic** [community/community#123849] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I was using jetbrains ide this morning and found that Copilot was not working.

Then I opened my Github Copilot settings page, and found no license status.

<img width="1124" alt="image" src="https://github.com/community/community/assets/12641585/6df602ba-3241-42d9-85d0-ad6ed1fb1ad1">

<img width="1420" alt="image" src="https://github.com/community/community/assets/12641585/a8c6f824-4dbe-4ad0-8fb4-6f3ebfdc92a8">

**Comment by gil-son** [comment, marked as answer]
Hello, how are you? I feel for that situation, you need to verify it


Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!

**Reply by ggggarlic** [reply]
Thank you very much, I have submitted a new ticket and expect it to be resolved. Thank you again.

**Reply by ghost** [reply]
Hi gil-son,
could you please also have a look at my ticket? https://support.github.com/ticket/personal/0/2784635
I have the same issue here.

Thanks

**Reply by ggggarlic** [reply]
It looks like the exception Copilot api call was caused by me using a local chat conversion tool. This is a violation of Copilot's terms of use. Please take this as a warning and respect github and the services provided by Copilot.

**Reply by gitHubhuanhuan** [reply]
官方给你答复了？我的还没给我答复

**Comment by gitHubhuanhuan** [comment]
问题解决了吗？我也有这个问题，正在等待解决~

**Reply by ggggarlic** [reply]
I posed my question in github support, and so far there has been no reply.

**Reply by gitHubhuanhuan** [reply]
我以为你是中国的呢
